linkis-engineconn-plugins/presto/src/test/scala/org/apache/linkis/engineplugin/presto/utils/testPrestoSQLHook.scala

```diff
@@ -0,0 +1,32 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.linkis.engineplugin.presto.utils
+
+import org.junit.jupiter.api.{Assertions, Test}
+
+class testPrestoSQLHook {
+
+  @Test
+  def testPreExecuteHook {
+    val prestoSQLHook = PrestoSQLHook
+    val code = "1104"
+    val codes = prestoSQLHook.preExecuteHook(code)
```

Comment:
Not cover the actual logic `org.apache.linkis.engineplugin.presto.utils.PrestoSQLHook#replaceBackQuoted`